client/components/payments-activity/payments-data-tile.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { formatCurrency } from 'wcpay/utils/currency';
+import Loadable from '../loadable';
+interface PaymentsDataTileProps {
+	id: string; // id of the element can be used for CSS styling
+	label: string; // The label appears as title of the tile
+	currencyCode: string; // The currency to be displayed in the tile
+	tooltip?: React.ReactElement; // For optionally passing the ClickTooltip component
+	amount?: number; // The amount to be displayed in the tile
+	isLoading?: boolean; // Loading state of the tile
+	reportLink?: string; // Optional hover link to view report
+}
```

Comment:
💭 If we want to annotate this "the TS way", we can add `/** */` comments like so.

This allows TS to see the annotations when using VSCode, for example when inspecting props with mouseover/cursor.

```suggestion
interface PaymentsDataTileProps {
	/**
	 * The id of the element can be used for CSS styling
	 */
	id: string;
	/**
	 * The label appears as title of the tile
	 */
	label: string;
	/**
	 * The currency to be displayed in the tile
	 */
	currencyCode: string;
	/**
	 * For optionally passing the ClickTooltip component
	 */
	tooltip?: React.ReactElement;
	/**
	 * The amount to be displayed in the tile
	 */
	amount?: number;
	/**
	 * Loading state of the tile
	 */
	isLoading?: boolean;
	/**
	 * Optional hover link to view report
	 */
	reportLink?: string;
}
```

Reply:
TIL - I actually wondered earlier why we would use a larger comment block. Now I know why!  Done here - https://github.com/Automattic/woocommerce-payments/pull/8564/commits/c7504cf187f3276ccecd836701f080906e4381a0